.github/workflows/main.yml

```diff
@@ -10,9 +10,11 @@ concurrency:
   cancel-in-progress: ${{ github.event_name == 'pull_request' }}
 
 env:
-  OPENAI_API_KEY: xxx
-  AZURE_OPENAI_API_HOST: xxx
   ANTHROPIC_API_KEY: xxx
+  AZURE_OPENAI_API_HOST: xxx
+  NEXT_PUBLIC_SUPABASE_ANON_KEY: xxx
+  NEXT_PUBLIC_SUPABASE_URL: https://placeholder.promptfoo.dev
+  OPENAI_API_KEY: xxx
```

Comment:
I prefer to preserve the original order, as the vast majority of people need to replace the OpenAI key more so than the other API keys.